src/Microsoft.Azure.WebJobs.ServiceBus/ServiceBusAccountAttribute.cs

```diff
@@ -31,5 +31,18 @@ public ServiceBusAccountAttribute(string account)
         /// Gets the name of the ServiceBus connection string to use.
         /// </summary>
         public string Account { get; private set; }
+
+        /// <inheritdoc />
+        string IConnectionProvider.Connection
```

Comment:
yeah, that will give us better impedence with the conversion back and forth from Functions.json. 

---
In reply to: [108546447](https://github.com/Azure/azure-webjobs-sdk/pull/1084#discussion_r108546447) [](ancestors = 108546447)